package.json

```diff
@@ -8,9 +8,7 @@
     "start": "next start",
     "lint": "next lint",
     "lint:fix": "next lint --fix",
-    "test:unit": "jest",
-    "cypress:open": "cypress open",
-    "test:e2e": "cypress run",
+    "test:unit": "jest --collectCoverage=false",
```

Comment:
todo: undo collectCoverage=false

For some reason when I run unit tests using Jest-runner in the IDE I;m getting coverage report which is obscuring  the test result. It's not helpful.